### PR TITLE
fix: Fix several missed logging statements

### LINKED ISSUE #1534: GET token-repository/projects/:id/tokens does not log error
It looks like the `GET token-repository/projects/:id/tokens` does not log anything on error.

Example from renkulab.io
GET http://renku-token-repository:9003/projects/20492/tokens

### PR DESCRIPTION
**Pull request by eikek**
Makes the `TestLogger` log in the effect and not as a side effect. Same for measuring and logging execution times.

Closes: #1534

### CODE REVIEW

File: event-log/src/test/scala/io/renku/eventlog/events/producers/cleanup/DispatchRecoverySpec.scala [hunk 1]
```diff
@@ -61,15 +61,15 @@ class DispatchRecoverySpec extends AnyWordSpec with should.Matchers with MockFac
...
 
-      dispatchRecovery.returnToQueue(event, sendingResults.generateOne) shouldBe ().pure[Try]
+      dispatchRecovery.returnToQueue(event, sendingResults.generateOne).unsafeRunSync() shouldBe ()
     }
   }
 
   "recovery" should {
 
-    "retry changing event status if status update failed initially" in new TestCase {
+    "reIO changing event status if status update failed initially" in new TestCase {
```
Comment (jachro) [inline event-log/src/test/scala/io/renku/eventlog/events/producers/cleanup/DispatchRecoverySpec.scala]: I guess a 'replaceAll' sort of issue.

File: event-log/src/test/scala/io/renku/eventlog/events/producers/triplesgenerated/DispatchRecoverySpec.scala [hunk 2]
```diff
@@ -61,15 +61,15 @@ class DispatchRecoverySpec extends AnyWordSpec with should.Matchers with MockFac
...
 
-      dispatchRecovery.returnToQueue(event, sendingResults.generateOne) shouldBe ().pure[Try]
+      dispatchRecovery.returnToQueue(event, sendingResults.generateOne).unsafeRunSync() shouldBe ()
     }
   }
 
   "recovery" should {
 
-    "retry changing event status if status update failed initially" in new TestCase {
+    "reIO changing event status if status update failed initially" in new TestCase {
```
Comment (jachro) [inline event-log/src/test/scala/io/renku/eventlog/events/producers/triplesgenerated/DispatchRecoverySpec.scala]: Same as above

File: token-repository/src/test/scala/io/renku/tokenrepository/repository/creation/TokensCreatorSpec.scala [hunk 3]
```diff
@@ -197,31 +196,31 @@ class TokensCreatorSpec extends AnyWordSpec with MockFactory with should.Matcher
...
       givenIntegrityCheckPasses(projectId, tokenCreationInfo.token, newTokenEncrypted)
 
       givenSuccessfulTokensRevoking(projectId, tokenCreationInfo, userAccessToken)
 
-      tokensCreator.create(projectId, userAccessToken) shouldBe ().pure[Try]
+      tokensCreator.create(projectId, userAccessToken).unsafeRunSync() shouldBe ()
     }
 
-    "retry if the token after storing couldn't be found" in new TestCase {
+    "reIO if the token after storing couldn't be found" in new TestCase {
```
Comment (jachro) [inline token-repository/src/test/scala/io/renku/tokenrepository/repository/creation/TokensCreatorSpec.scala]: One more

File: token-repository/src/test/scala/io/renku/tokenrepository/repository/creation/TokensCreatorSpec.scala [hunk 4]
```diff
@@ -244,22 +243,27 @@ class TokensCreatorSpec extends AnyWordSpec with MockFactory with should.Matcher
...
+        intercept[Exception] {
+          tokensCreator.create(projectId, userAccessToken).unsafeRunSync()
+        }
 
-      tokensCreator.create(projectId, userAccessToken) shouldBe exception
+      ex shouldBe exception
     }
 
-    "fail if retry process hit the max number of attempts" in new TestCase {
+    "fail if reIO process hit the max number of attempts" in new TestCase {
```
Comment (jachro) [inline token-repository/src/test/scala/io/renku/tokenrepository/repository/creation/TokensCreatorSpec.scala]: one more

**Review by jachro — APPROVED** [review]

File: token-repository/src/test/scala/io/renku/tokenrepository/repository/creation/TokensCreatorSpec.scala (hunk 4 above)
Comment (eikek) [reply token-repository/src/test/scala/io/renku/tokenrepository/repository/creation/TokensCreatorSpec.scala]: Oh no, thank you! That search&replace went a bit too far :)

**Pull request merged**